langchain/src/smith/runner_utils.ts

```diff
@@ -25,7 +25,7 @@ import {
 } from "langsmith";
 import { EvaluationResult, RunEvaluator } from "langsmith/evaluation";
 import { DataType } from "langsmith/schemas";
-import type { TraceableFunction } from "langsmith/traceable";
+import type { TraceableFunction } from "langsmith/singletons/traceable";
```

Comment:
No worries if it's already done, but seems like an unnecessary breaking change - why move this?